clients/extension/scripts/popup.js

```diff
@@ -171,7 +177,7 @@ async function initPopup() {
   });
 
   optionsFormEl.addEventListener('change', () => {
-    persistSettings();
```

Comment:
counter-proposal: move the reading the settings from the dom part to a separate function `readSettingsFromDOM` and use that in `click` handler so we don't need to make this mistake that took me a while to notice even with the explicit PR to fix it 😆 

this works too though :)